### DISCUSSION TITLE
posts/2024-04-03-ai-antibiotics/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#12] [Announcements]
# posts/2024-04-03-ai-antibiotics/

Bacterial resistance to existing antibiotics is an urgent threat. AI is being used to search for new antibiotics.

https://rachel.fast.ai/posts/2024-04-03-ai-antibiotics/

**Comment by yvesnewman** [comment]
Again, awesome summary and a huge fan of your work! Keep it up

**Reply by racheltho** [reply]
Thank you, I appreciate it!

**Comment by TinyAnts** [comment]
Huge fan of your work and gave been following for a several years! I am a vet student and a post graduate AI/ML engineer! I am so interested in this topic as antibiotic resistance is a such a huge problem in veterinary medicine and almost nobody takes any steps as there is less money in it than human medicine sadly! But I want to make change and I think I will dig deeper if there is a potential in pursuing this topic deeply as there is a clear problem with urgency! 

If there is something I can do to help contribute like a open source tool/dataset/library (just like fast ai) I would love to help as I understand the medicine and AI...
